### PR TITLE
Added support for mongoimport and mongoexport.

### PR DESCRIPTION
**Pull request by hoang1417**
As discussed in this #5, I made this new pull request to add `support for mongoimport and mongoexport`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
-0.4.1 (Next)
+0.4.2 (Next)
 ============
 
 * Your contribution here.
 
+0.4.1 (10/23/2015)
```
Comment (dblock) [inline CHANGELOG.md]: This one is still wrong. It should just be 0.4.1 (next), you're not doing a release ;)

Comment (dblock) [inline CHANGELOG.md]: You can amend this via git commit --amend and force push your topic branch and it will update here.

**Comment by hoang1417** [comment]
@dblock, okay got it, thanks. Hope it's the last thing needed to update. :smile:.

**Comment by dblock** [comment]
Not really, but I tortured you enough :) 

Merged via https://github.com/dblock/mongoid-shell/commit/d3cb93bff342fded925340453161834671505894.

Thanks!

**Comment by dblock** [comment]
FYI I cut 0.4.1 with this code.

**Comment by hoang1417** [comment]
Oh I see the missing piece (date of 0.4.1) . That's my bad, thanks for correcting it and other issues. :)

**Comment by dblock** [comment]
No no, I added that when I released 0.4.1. If you look at the commit I made it did this:

screenshot 2015-10-26 06 47 59

**Comment by hoang1417** [comment]
I see. :disappointed: Then the mistake is to add the `0.4.2` too soon before your release :smile:.

**Comment by dblock** [comment]
@hoang1417 No big deal! Thanks so much for contributing.

**Pull request closed without merging**